src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -312,6 +308,55 @@ void testCreateElectionsForDarCollection() throws Exception {
     verify(darCollectionDAO, times(1)).findDARCollectionByCollectionId(any());
   }
 
+  @Test
+  void testCreateElectionsForDarCollectionEmpty() {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+
+    assertThrows(IllegalStateException.class, () -> {
+      service.createElectionsForDarCollection(user, collection);
+    });
+  }
+
+  @Test
+  void testCreateElectionsForDarCollectionNull() throws Exception {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+    when(darCollectionServiceDAO.createElectionsForDarCollection(any(), any())).thenReturn(null);
+
+    assertThrows(IllegalStateException.class, () -> {
+      service.createElectionsForDarCollection(user, collection);
+    });
+  }
+
+  @Test
+  void testCreateElectionsForDarCollectionVoteUsersException() throws Exception {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+    when(darCollectionServiceDAO.createElectionsForDarCollection(any(), any())).thenReturn(
+        List.of("electionId"));
+    when(voteDAO.findVoteUsersByElectionReferenceIdList(any())).thenThrow(
+        IllegalArgumentException.class);
+
+    service.createElectionsForDarCollection(user, collection);
+    verify(darCollectionServiceDAO, times(1)).createElectionsForDarCollection(any(), any());
+    verify(voteDAO, times(1)).findVoteUsersByElectionReferenceIdList(any());
+    verify(emailService, times(0)).sendDarNewCollectionElectionMessage(any(), any());
```

Comment:
If `emailService` isn't used at all, I would use `verifyNoInteractions()` instead
```suggestion
    verifyNoInteractions(emailService);
```